library/VTKExtensions/Rendering/glsl/vtkF3DBitonicSortLocalDisperseCS.glsl

```diff
@@ -0,0 +1,29 @@
+#version 430
+
+//VTK::BitonicDefines::Dec
+
+layout(local_size_x = WorkgroupSize) in;
+layout(std430) buffer;
+
+layout(binding = 0) buffer Keys
+{
+  KeyType key[];
+};
+
+layout(binding = 1) buffer Values
+{
+  ValueType value[];
+};
+
+layout(location = 0) uniform int count;
+
+//VTK::BitonicFunctions::Dec
+
+void main()
+{
+  for (uint h = WorkgroupSize; h >= 1; h /= 2)
```

Comment:
I'm not sure that height == 1 is expected to append for local disperse, for me the condition should be `h > 1` here 🤔

Reply:
understood! thanks :)